util-hadoop/pom.xml

```diff
@@ -97,6 +92,16 @@
       <artifactId>flogger-system-backend</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud.bigdataoss</groupId>
+      <artifactId>util</artifactId>
+      <version>${bigdataoss.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.cloud.bigdataoss</groupId>
+      <artifactId>gcsio</artifactId>
```

Comment:
Hadoop utils can not depend on GCSIO.
Why is this necessary?

Reply:
Because AccessTokenProvider inherits from StorageAccessTokenProvider which exists in GCSIO